resources/views/company/index.blade.php

```diff
@@ -144,12 +146,13 @@
             });
 
             // Edit Company
+            let id = null
```

Comment:
what is the purpose of this global variable?

Reply:
![image](https://github.com/user-attachments/assets/d1674d4d-0f4f-4760-95d5-b3d0a66a3230)


im getting the id from the parent row of the button that's clicked and i need the id again to submit the form so to get the id out the scope im setting it as a global variable

i have attached ss as a visual explanation please refer @Raza9798